rubygem-smart_proxy_ansible/rubygem-smart_proxy_ansible.spec

```diff
@@ -7,15 +7,16 @@
 
 Summary: Ansible support for Foreman smart proxy
 Name: rubygem-%{gem_name}
-Version: 1.0.0
+Version: 1.0.1
 Release: 1%{?foremandist}%{?dist}
 Group: Applications/System
 License: GPLv3
 URL: https://github.com/theforeman/smart_proxy_ansible
 Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
 
 Requires: %{?rhel:tfm-}rubygem(smart_proxy_dynflow_core) >= 0.1.5
-Requires: %{?rhel:tfm-}rubygem(foreman_ansible_core)
+Requires: %{?rhel:tfm-}rubygem(foreman_ansible_core) >= 1.0.0
+Requires: %{?rhel:tfm-}rubygem(foreman_ansible_core) < 2.0.0
```

Comment:
@domcleal Yeah - the one in foreman_ansible itself is too strict by mistake, >= 1.0 is just fine